src/Components/Common/components/ButtonV2.tsx

```diff
@@ -53,6 +54,14 @@ export type ButtonProps = RawButtonProps &
      * Whether the button should be disabled and show a loading animation.
      */
     loading?: boolean | undefined;
+    href?: string | undefined;
+    /**
+     * A button will convert to a link if the `href` prop is set.
+     */
+    target?: string | undefined;
+    /**
+     * Link target. Only applicable if `href` is set.
+     */
```

Comment:
Doc comments come above the intended line so that when we hover on 'href' in the editor, it shows up.
```suggestion
    /**
     * A button will convert to a link if the `href` prop is set.
     */
    href?: string | undefined;
    /**
     * Link target. Only applicable if `href` is set.
     */
    target?: string | undefined;
```

Reply:
Whoops, thanks for pointing that out 😀